src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,10 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @password_changable = @configuration['change_password'] && CONFIG['proxy_auth_mode'] != :on && CONFIG['ldap_mode'] != :on
```

Comment:
I agree, I'll implement a method to the model. Regarding your question: The `@configuration` instance variable holds a object to the first record of the `Configuration` model. As you can see in the `Configuration` model (https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/configuration.rb#L21) the `OPTIONS_YML` variable seems to be a merge of `CONFIG` (https://github.com/openSUSE/open-build-service/blob/master/src/api/config/environment.rb#L9) and the data of the record. So it seems that `@configuration['change_password']` is the same like `CONFIG['change_passwd']`. But I would not waste time with this since Adrian said (in a private discussion) we'll drop this configuration option anyways.